lib/widgets/witnet/transactions/value_transfer/create_dialog_box/vtt_builder/02_select_miner_fee.dart

```diff
@@ -15,6 +15,7 @@ import 'package:my_wit_wallet/widgets/clickable_box.dart';
 import 'package:my_wit_wallet/widgets/input_amount.dart';
 import 'package:my_wit_wallet/widgets/toggle_switch.dart';
 import 'package:font_awesome_flutter/font_awesome_flutter.dart';
+import 'package:flutter_gen/gen_l10n/app_localizations.dart';
```

Comment:
Add localization for estimated options label (Stinky, Low, Medium, High, Opulent, Custom), to do so, you need to update the way the estimations copies are shown in the ui:
`        EstimatedFeeOptions label =
            _minerFeeOptionsNanoWit.keys.toList()[index];`